hive/src/main/java/com/netease/arctic/hive/utils/HivePartitionUtil.java

```diff
@@ -72,4 +78,84 @@ public static Partition newPartition(
     }
     return p;
   }
+
+  public static Partition getPartition(HMSClient hmsClient,
+                                       ArcticTable arcticTable,
+                                       List<String> partitionValues,
+                                       String partitionLocation) {
+    String db = arcticTable.id().getDatabase();
+    String tableName = arcticTable.id().getTableName();
+    Table hiveTable;
+    try {
+      hiveTable = hmsClient.run(c -> c.getTable(db, tableName));
```

Comment:
Why dose getting table first needed?
Getting partition will throw exception either when table not existed.

Reply:
OK, I will remove it